src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Queries/Search/SearchDialogQuery.cs

```diff
@@ -181,8 +182,7 @@ public async Task<SearchDialogResult> Handle(SearchDialogQuery request, Cancella
         {
             foreach (var seenRecord in paginatedList.Items.SelectMany(x => x.SeenSinceLastUpdate))
             {
-                seenRecord.IsCurrentEndUser = seenRecord.SeenBy.ActorId == request.EndUserId;
-
+                seenRecord.IsCurrentEndUser = seenRecord.SeenBy.ActorId == NorwegianPersonIdentifier.PrefixWithSeparator + request.EndUserId;
```

Comment:
EndUserId is already prefixed as per input validation rules

Reply:
DOH 🤦🏼